topgun/k8s/baggageclaim_drivers_test.go

```diff
@@ -46,15 +49,21 @@ var _ = Describe("baggageclaim drivers", func() {
 			It("successfully recreates the worker", func() {
 				By("deploying concourse with ONLY one worker and having the worker pod use the gcloud disk and format it with btrfs")
 
+				scName := "btrfs"
+				createBtrfsStorageClass(scName)
+
 				setReleaseNameAndNamespace("real-btrfs-disk")
+				pvcName := "disk-" + namespace
 
 				deployWithDriverAndSelectors("btrfs", UBUNTU,
 					"--set=persistence.enabled=false",
 					"--set=worker.additionalVolumes[0].name=concourse-work-dir",
-					"--set=worker.additionalVolumes[0].gcePersistentDisk.pdName=disk-topgun-k8s-btrfs-test",
-					"--set=worker.additionalVolumes[0].gcePersistentDisk.fsType=btrfs",
+					"--set=worker.additionalVolumes[0].persistentVolumeClaim.claimName="+pvcName,
 				)
 
+				// We create the PVC after deploying because we need the namespace to be created first by helm
+				createPVC(pvcName, scName)
```

Comment:
I might not remember correctly, but didn't we use to get the PVC created when deploying the chart? Has something changed here or am I missing something?

Reply:
This test requires a volume that is using `btrfs`. It expected the volume to already be created and available. However, the volume didn't exist on the topgun cluster, we believe it might have been deleted by mistake. The commit message has some more details.

Reply:
I think something along the lines of having the cluster already bootstrapped with the storage class. We are considering a similar approach [here](https://github.com/concourse/greenpeace/blob/a2e1ff274c57f1166a1eae4a385e818e3c041cc0/terraform/production/cluster.tf#L32-L42).